media/CfgEditor/displaycfg.js

```diff
@@ -141,6 +141,10 @@ export function displayCfgToEditor(cfg) {
   );
   
   // TODO Support one-import-bcq
+  const oneImportEdgeTPU = cfg["one-import-edgetpu"];
```

Comment:
Is this part is for `one-import-bcq`? I think it should be in the part of `one-import-edgetpu`.
Please change the location :)

Reply:
Done! thank you! :)